src/secret_sharing/replicated/semi_honest/mod.rs

```diff
@@ -0,0 +1,3 @@
+mod additive_share;
+
+pub use additive_share::AdditiveShare;
```

Comment:
A no-op mod.rs probably isn't helping much here.

You could drop the module and even the subdirectory and have malicious.rs and semi_honest.rs in the replicated/ direcory.

Reply:
I'm going to add xor_share.rs under replicated/semi_honest/. Not sure whether we'll have the malicious version of it, but it'll be nice to have a consistent directory structure?